### DISCUSSION TITLE
How make a query like this:

### DISCUSSION THREAD

**Discussion by Pablete01** [supabase-community/supabase-kt#132] [Q&A] [answered]
Hi.
I need to make a query like this:
Latitude between val min and val max.
Longitude between val min and val max
I try:
.... select{
Lt("latitude", val min)
Gt("latitude, val max)
Lt("longitude, val min)
Gt('longitude, val max)
}

But doesn't work. I think this is not the correct format.
Could please help me with this trouble?
Thanks

**Comment by jan-tennert** [comment, marked as answer]
Can you try this?
```kotlin
select {
    and {
        lt("latitude", min)
        gt("latitude", max)
    }
    and { 
        lt("longitude", min)
        gt("longitude", max)
    }
}
```

**Comment by Pablete01** [comment]
Thanks!
